helpers/z_schema.js

```diff
@@ -208,7 +208,7 @@ var liskFormats = {
 	 * @todo Add description for the function, the params and the return value
 	 */
 	queryList(obj) {
-		if (obj == null || typeof obj !== 'object' || _.isArray(obj)) {
+		if (!_.isObject(obj) || _.isArray(obj)) {
```

Comment:
@diego-G can you replace _.isArrays to Array.isArray ?

lodash function does the following check:
```javascript
function isArrayLike(value) {
      return value != null && isLength(value.length) && !isFunction(value);
    }
```
That's why I prefer to use native is array function.

Reply:
@MaciejBaj @diego-G just fyi guys, `obj === Object(obj)` this check is behaving as you expected. See: https://stackoverflow.com/a/14706877/1417536 

underscore.js was using this solution.

Reply:
I'm also ok to create another issue for these usages If you agree to change these.

Reply:
Yes, please. I prefer to open another issue for that.